docs/reference-spec.md

```diff
@@ -8,9 +8,9 @@ A bindle has a name. For example, if we were to write a `hello world` bindle, it
 
 When a bindle is concretized into an invoice, it MUST have a name and a version. Version MUST be a valid [SemVer 2 version](https://semver.org).
 
-A bindle name can be one or more characters from the set of unicode alphabetic and numeric characters, as well as the characters `_` and `/`.
+A bindle name can be one or more characters from the set of unicode alphabetic and numeric characters, as well as the characters `-`, `.`, `_` and `/`.
```

Comment:
Bit of a drive-by update but seems obvious to me; `.` is used in examples right below and `-` is allowed in domain names.